This file is part of MXE.
See index.html for further information.

Fix flann.pc to link with the static libraries.
diff -uNr flann-1.7.1-src/cmake/flann.pc.in flann-1.7.1-src.patched/cmake/flann.pc.in
--- flann-1.7.1-src/cmake/flann.pc.in	2011-12-23 12:16:09.000000000 +0100
+++ flann-1.7.1-src.patched/cmake/flann.pc.in	2012-10-01 12:46:31.617169719 +0200
@@ -8,6 +8,6 @@
 Description: @PKG_DESC@
 Version: @FLANN_VERSION@
 Requires: @PKG_EXTERNAL_DEPS@
-Libs: -L${libdir} -lflann_cpp -lflann_cpp-gd
+Libs: -L${libdir} -lflann_cpp_s -lflann_cpp_s-gd
 Cflags: -I${includedir}
 

